tst/testinstall/pgroups.tst

```diff
@@ -101,16 +101,12 @@ gap> K := Group((1,3),(1,2,3,4),(5,6,7));
 Group([ (1,3), (1,2,3,4), (5,6,7) ])
 gap> IsNilpotentGroup(K);
 true
-gap> HasIsPGroup(K);
-true
 gap> IsPGroup(K);
 false
 gap> L := Group((2,4), (1,2,3,4));
 Group([ (2,4), (1,2,3,4) ])
 gap> IsNilpotentGroup(L);
 true
-gap> HasIsPGroup(L) and HasPrimePGroup(L);
-true
```

Comment:
These two `HasIsPGroup` are meant to check that the "right" method for `IsNilpotentGroup` is called (and that methods has a static rank adjustment of +20)... but that seems pointless, so let's just make the tests less specific.